assets/js/src/form_editor/blocks/input_styles_settings.jsx

```diff
@@ -1,16 +1,61 @@
-import React from 'react';
+import React, { useState } from 'react';
 import MailPoet from 'mailpoet';
 import {
   Panel,
   PanelBody,
+  ToggleControl,
 } from '@wordpress/components';
+import { partial } from 'underscore';
+import PropTypes from 'prop-types';
 
-const InputStylesSettings = () => (
-  <Panel>
-    <PanelBody title={MailPoet.I18n.t('formSettingsStyles')} initialOpen={false}>
-      TODO: styles settings
-    </PanelBody>
-  </Panel>
-);
+const InputStylesSettings = ({
+  styles,
+  onChange,
+}) => {
+  const [localStyles, setStyles] = useState(styles);
 
-export default InputStylesSettings;
+  const updateStyles = (property, value) => {
+    const updated = { ...localStyles };
+    updated[property] = value;
+    onChange(updated);
+    setStyles(updated);
+  };
+  return (
+    <Panel>
+      <PanelBody title={MailPoet.I18n.t('formSettingsStyles')} initialOpen={false}>
+        <ToggleControl
+          label={MailPoet.I18n.t('formSettingsDisplayFullWidth')}
+          checked={localStyles.fullWidth}
+          onChange={partial(updateStyles, 'fullWidth')}
+        />
+        <ToggleControl
+          label={MailPoet.I18n.t('formSettingsInheritStyleFromTheme')}
+          checked={localStyles.inheritFromTheme}
+          onChange={partial(updateStyles, 'inheritFromTheme')}
+        />
+        {!localStyles.inheritFromTheme ? (
+          <>
```

Comment:
Why do we need the fragment here?

Reply:
There are more elements in this jsx block so we need to wrap them.